mithril-signer/src/services/aggregator_client.rs

```diff
@@ -214,36 +205,6 @@ impl AggregatorClient for AggregatorHTTPClient {
         }
     }
 
```

Comment:
The /certificate_pending still exist in openapi.yaml even if it was deprecated. 
Is it normal to remove all tests ? What happen when the route is called ?